src/Symfony/Component/Notifier/Notification/WebPushNotificationInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Notification;
+
+use Symfony\Component\Notifier\Message\WebPushMessage;
+use Symfony\Component\Notifier\Recipient\RecipientInterface;
+
+/**
+ * @experimental in 5.3
+ */
```

Comment:
Unfortunately we cannot introduce sth experimental in the last minor before an LTS release

Reply:
Leftover from the PR start :)

Reply:
Thanks